packages/taiko-client/proposer/transaction_builder/blob.go

```diff
@@ -83,31 +90,65 @@ func (b *BlobTransactionBuilder) Build(
 	signature[64] = uint8(uint(signature[64])) + 27
 
 	var (
-		to   = &b.taikoL1Address
-		data []byte
+		to            = &b.taikoL1Address
+		data          []byte
+		encodedParams []byte
```

Comment:
We don't need to decode `encodedParams`, using it as an interface is ok when calling `abi.Pack`.

Reply:
don't think we are decoding anything here, no?